docs/xls-33d-multi-purpose-tokens/reference/mptokenissuance.md

```diff
@@ -0,0 +1,84 @@
+---
+html: mptokenissuance.html
+parent: basic-data-types.html
+blurb: Introduction to XRPL MPTs.
+labels:
+  - Multi-purpose Tokens, MPTs, Tokens
+---
+# MPTokenIssuance
+
+The `MPTokenIssuance` object represents a single MPT issuance and holds data associated with the issuance itself. Token issuances are created using the `MPTokenIssuanceCreate` transaction and can be destroyed by the `MPTokenIssuanceDestroy` transaction.
+
+_(Added by the [MPTokenV1_1 amendment][].)_
+
+## Example {{currentpage.name}} JSON
+
+```json
+{
+    "LedgerEntryType": "MPTokenIssuance",
+    "Flags": 131072,
+    "Issuer": "rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW",
+    "AssetScale": "2",
+    "MaximumAmount": "100000000",
+    "OutstandingAmount": "5",
```

Comment:
There is nothing wrong with this JSON. But want to bring something up since there is something confusing here.

One caveat is that for fields that have type `Uint64` (`MaximumAmount`, `OutstandingAmount`, `MPTokenAmount`), these fields are actually **hex-encoded strings**, not number.

If `OutstandingAmount` wants to store a value of `1000` in decimal, it would actually return this in JSON:
  ```
"OutstandingAmount": "3e8"
```

On the other hand, for fields that have `Amount` type with curly braces( eg. `Amount`, `delivered_amount`), the values inside are always **decimal strings**

If we specify a Payment transaction to send a payment of `1000` tokens, it would look like
  ```json
"Amount": {
   "mpt_issuance_id":"123.....",
   "value":"1000", // notice this is not hex-encoded!
}
```


This would be very confusing to users (unfortunately couldn't be changed). So i thought it would be useful to document this somewhere that is easily accessible

Reply:
Good that you pointed this out. I added this discussion to `mptoken.md`.